src/pages/FavouriteMovie.js

```diff
@@ -55,10 +65,10 @@ class FavouriteMovie extends Component {
           </button>
         </div>
 
-        {this.state.movie_data.map((i) => (
-          <div className="div1">
-            <h4 className="par">{i.Title}</h4>
-            <img src={i.Poster} id="imageMovie"></img>
+        {this.state.movie_data.map((i, x1, x2) => (
+          <div key={x1} className="div1">
+            <h4 key={x2} className="par">{i.Title}</h4>
+            <img key={x1} src={i.Poster} id="imageMovie"></img>
```

Comment:
No need to add keys here, only parent element that has may identical siblings should have a unique key